libraries/core/src/test/kotlin/com/bumble/appyx/core/lifecycle/MinimumCombinedLifecycleTest.kt

```diff
@@ -39,10 +39,10 @@ class MinimumCombinedLifecycleTest {
 
     @Test
     fun `destroyed when one of lifecycles is destroyed`() {
-        val test1 = TestLifecycle()
-        test1.state = State.DESTROYED
-        val test2 = TestLifecycle()
+        val test1 = TestLifecycle(State.RESUMED)
```

Comment:
test changes due to behaviour change in https://developer.android.com/jetpack/androidx/releases/lifecycle#2.5.0-alpha03